src/current/v23.1/sso-sql.md

```diff
@@ -102,33 +100,30 @@ You can also view all of your cluster settings in the DB Console.
     SET CLUSTER SETTING server.jwt_authentication.audience = '984901724939-njig7lkv7k724rbv2hllvr4of8ul7th7.apps.googleusercontent.com';
     ~~~
 
-1. `server.jwt_authentication.claim`
-    This setting determines which field in the JWT your cluster will use to the identity to a SQL user. The value of `email` indicates the email field in the JWT.
+1. Set `server.jwt_authentication.claim` to the field in the JWT that the cluster will use as the identity of the SQL user. To use the email field in the JWT, set `server.jwt_authentication.claim` to `email`.
```

Comment:
'SQL user' could be a link to https://www.cockroachlabs.com/docs/cockroachcloud/managing-access#manage-sql-users-on-a-cluster